src/quantile.js

```diff
@@ -1,10 +1,19 @@
-import ascending from "./ascending.js";
+import max from "./max.js";
+import min from "./min.js";
+import quickselect from "./quickselect.js";
 import number, {numbers} from "./number.js";
 
-export default function quantile(values, p, valueof) {
+export default function quantile(values, p, valueof = number) {
```

Comment:
This default value `= number` isn’t needed because `numbers(values, undefined)` will be equivalent, no? I’ll try it and see.